"use strict"

try {
    //x = 10;
    miFuncion();
} catch (error) {
    console.log(error);
    
}
finally{
    console.log('Termina la revision  del codigo');
}

console.log('Continuamos');
